hs_core/hydroshare/utils.py

```diff
@@ -222,17 +221,16 @@ def get_temp_dir():
 
 
 # TODO: should be ResourceFile.replace
-def replace_resource_file_on_irods(new_file, original_resource_file, user):
+def replace_resource_file_on_s3(new_file, original_resource_file, user):
     """
-    Replaces the specified resource file with file (new_file) by copying to iRODS
-    (local or federated zone)
-    :param new_file: file path for the file to be copied to iRODS
+    Replaces the specified resource file with file (new_file) by copying to S3
+    :param new_file: file path for the file to be copied to S3
     :param original_resource_file: an instance of ResourceFile that is to be replaced
     :param user: user who is replacing the resource file.
     :return:
     """
     ori_res = original_resource_file.resource
-    istorage = ori_res.get_irods_storage()
+    istorage = ori_res.get_s3_storage()
```

Comment:
do we still want to call it `istorage`???

Reply:
I went back and forth on this. I think istorage still makes sense as it as the i could stand for interface so I left it.

Reply:
sounds good